src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_0F_LOTTERIES = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball[] balls = new Ball[NUMBER_0F_LOTTERIES];
+
+        for (Ball randomBall : balls) {
+            randomBall = lottery.getRandomBall();
+            System.out.println("The ball number: " + randomBall.getNumber()
```

Comment:
```suggestion
            System.out.println(randomBall);
```
якщо перевизначити коректно toString() метод - можна буде просто передати об'єкт в system out